backend/src/main/java/fk/prof/backend/worker/BackendDaemon.java

```diff
@@ -73,14 +78,15 @@ private AggregationWindowPlannerStore buildAggregationWindowPlannerStore() {
     return new AggregationWindowPlannerStore(
         vertx,
         configManager.getBackendId(),
-        config().getInteger("aggregation.window.duration.mins", 30),
+        config().getInteger("aggregation.window.duration.secs", 1800),
         config().getInteger("aggregation.window.end.tolerance.secs", 120),
         config().getInteger("policy.refresh.offset.secs", 300),
         config().getInteger("scheduling.buffer.secs", 30),
         config().getInteger("work.assignment.max.delay.secs", 120),
         workSlotPool,
         activeAggregationWindows,
-        this::getWorkFromLeader);
+        this::getWorkFromLeader,
+        this::serializeAndPersistAggregationWindow);
```

Comment:
Eventually, feedback from S3 GW should be one of the things that lead to backend projecting bad-health in /leader/load rpc. Because this is important for cluster to self-heal in scenarios where backend is partitioned away from S3 GW.